src/layouts/Dashboard/components/ReviewRequestModal/RequestOverview/RequestOverview.tsx

```diff
@@ -0,0 +1,335 @@
+          {time}
+        </Text>
+      </VStack>
+    </HStack>
+  )
+}
+
+export interface RequestOverviewProps {
+  items: EditedItemProps[]
+}
+
+export const RequestOverview = ({
+  items,
+}: RequestOverviewProps): JSX.Element => {
+  const {
+    isExpanded,
+    inputRef,
+    handleExpansion,
+    handleCollapse,
+  } = useSearchbar({})
+  const columnHelper = createColumnHelper<EditedItemProps>()
+
+  const columns = [
+    columnHelper.display({
+      id: "filter",
+      header: () => (
+        <Th w="3rem">
+          <IconButton
+            aria-label="sort by file type"
+            icon={<BiFilterAlt />}
+            variant="link"
+          />
+        </Th>
+      ),
+      cell: (props) => getIcon(props.row.original.type),
+    }),
+    columnHelper.accessor((row) => `${row.name} ${row.path}`, {
+      id: "itemName",
+      cell: ({ row }) => (
+        <ItemName name={row.original.name} path={row.original.path} />
+      ),
+      header: () => (
+        <Th textTransform="capitalize">
+          <Text textStyle="subhead-2" textColor="text.label">
+            Item name
+          </Text>
+        </Th>
+      ),
+    }),
+    columnHelper.accessor(
+      (row) => `${row.lastEditedBy} ${row.lastEditedTime}`,
+      {
+        id: "lastEdited",
+        header: () => (
+          <Th textTransform="capitalize" w="12.5rem">
+            <HStack spacing="0.25rem">
+              <Text textStyle="subhead-2" textColor="text.label">
+                Last edited
+              </Text>
+              <IconButton
+                icon={<BiSort fontSize="1rem" />}
+                aria-label="view file changes"
+                variant="link"
+              />
+            </HStack>
+          </Th>
+        ),
+        cell: ({ row }) => (
+          <LastEditedMeta
+            lastEditedBy={row.original.lastEditedBy}
+            lastEditedTime={row.original.lastEditedTime}
+          />
+        ),
+      }
+    ),
+    columnHelper.display({
+      id: "actions",
+      header: () => <Th w="10rem" />,
+      cell: ({ row }) => (
+        <HStack spacing="0.25rem">
+          <Link href={row.original.url}>
+            <IconButton
+              icon={<BiGitCompare />}
+              aria-label="view file on staging"
+              variant="link"
+            />
+          </Link>
+          <IconButton
+            icon={<BiShow />}
+            aria-label="view file changes"
+            variant="link"
+          />
+        </HStack>
+      ),
+    }),
+  ]
+
+  const table = useReactTable({
+    data: items,
+    columns,
+    getCoreRowModel: getCoreRowModel(),
+  })
+
+  return (
+    <VStack spacing="1rem" mt="1rem">
+      <Flex w="100%">
+        <Box>
+          <Text textStyle="subhead-1" textColor="text.body">
+            <b>{items.length}</b> item(s) for review
+          </Text>
+          <Text textStyle="caption-2" textColor="text.placeholder">
+            Items below have been identified to have changes
```

Comment:
I think "Items" can be changed to "Item(s)" to match the title.

Reply:
can, good catch!